doc/rbd/nvmeof-target-configure.rst

```diff
@@ -0,0 +1,115 @@
+================================================
+Installing and Configuration the NVMe-oF Targets
+================================================
+
+Traditionally, block-level access to a Ceph storage cluster has been limited to
+QEMU and ``librbd``, which is a key enabler for adoption within OpenStack
+environments. Starting with the Ceph Reef release, block-level access has been 
+expanded to offer standard NVMe/TCP support, allowing wider platform usage and
+potentially opening new use cases.
+
+Prerequisites
+=============
+
+-  Red Hat Enterprise Linux/CentOS 8.0 (or newer); Linux kernel v4.16 (or newer)
+
+-  A working Ceph Storage cluster, deployed with ``cephadm``
+
+-  NVMe-oF gateways nodes, which can either be colocated with OSD nodes or on dedicated nodes
+
+-  Separate network subnets for NVME-oF front-end traffic and Ceph back-end traffic
+
+Explanation
+===========
+
+The Ceph NVMe/-oF gateway is both an NVMe-oF target and a Ceph client. Think of
+it as a "translator" between Ceph's RBD interface and the NVME-oF protocol. The
+Ceph NVMe-oF gateway can run on a standalone node or be colocated with other
+daemons, for example on a Ceph Object Store Disk (OSD) node. When colocating
+the Ceph NVMe-oF gateway with other daemons, ensure that sufficient CPU and
+memory are available. The steps below explain how to install and configure the
+Ceph NVMe/TCP gateway for basic operation.
+
+Requirements
+============
+
+-  A running Ceph Reef or later storage cluster
+
+-  Red Hat Enterprise Linux/CentOS 8.2 (or newer)
```

Comment:
There is a Prerequisites section above that says the same thing with regards to the storage cluster but lists a different RHEL version.  Once versions are clarified, I think this section should be removed.

Reply:
This section has been removed from the text.